phpBB/phpbb/forum/controller/viewforum.php

```diff
@@ -56,23 +71,33 @@ class viewforum
 	protected $parameters = [];
 
 	public function __construct(
+		\phpbb\auth\auth $auth,
 		\phpbb\config\config $config,
+		\phpbb\content_visibility $content_visibility,
 		\phpbb\controller\helper $helper,
 		\phpbb\event\dispatcher_interface $dispatcher,
+		\phpbb\forum\data_transformer $data_transformer,
 		\phpbb\forum\forum_retriever $forum_retriever,
 		\phpbb\language\language $language,
+		\phpbb\pagination $pagination,
 		\phpbb\forum\render\helper $render_helper,
 		request_interface $request,
+		\phpbb\template\template $template,
 		\phpbb\user $user,
 		\phpbb\forum\visibility_helper $visibility_helper)
 	{
+		$this->auth = $auth;
 		$this->config = $config;
+		$this->content_visibility = $content_visibility;
 		$this->controller_helper = $helper;
 		$this->dispatcher = $dispatcher;
+		$this->forum_data_transformer = $data_transformer;
 		$this->forum_retriever = $forum_retriever;
 		$this->language = $language;
 		$this->render_helper = $render_helper;
+		$this->pagination = $pagination;
 		$this->request = $request;
+		$this->template = $template;
 		$this->user = $user;
 		$this->visibility_helper = $visibility_helper;
```

Comment:
Couldn't find anything about it in the coding guidelines, but I would say it makes more sense to have the closing `)` on a new line. 
_(Same for all other constructors)_